bentoctl/cli/__init__.py

```diff
@@ -77,36 +79,34 @@ def init(file):
                 f"{file.relative_to(Path.cwd())}[/]"
             )
         deployment_config = DeploymentConfig.from_file(file)
-        deployment_config.generate()
+        if generate:
+            deployment_config.generate()
     except BentoctlException as e:
         console.print(f"[red]{e}[/]")
         sys.exit(1)
 
 
 @bentoctl.command()
-@click.option("--deployment_config_file", "-f", help="path to deployment_config file")
-def generate(deployment_config_file):
+@click.option("--deployment-config-file", "-f", default="deployment_config.yaml", help="path to deployment config file")
+@click.option("--values-only", is_flag=True, help="only update values")
+def generate(deployment_config_file, values_only):
     """
     Generate the files for the deployment based on the template type provided.
     """
     deployment_config = DeploymentConfig.from_file(deployment_config_file)
-    deployment_config.generate()
+    deployment_config.generate(values_only)
 
 
 @bentoctl.command()
 @click.argument("bento_tag")
 @click.option("--deployment_config_file", "-f", help="path to deployment_config file")
 @click.option("--push", is_flag=True, default=False)
-@click.option("--overwrite-deployable/--no-overwrite-deployable", default=True)
-@click.option("--registry-password")
-@click.option("--registry-username")
+@click.option("--debug", is_flag=True, default=False)
 def build(
     bento_tag,
     deployment_config_file,
     push,
-    overwrite_deployable,
-    registry_username,
-    registry_password,
+    debug,
```

Comment:
we already have debug functionality setup for all commands. Its sets an env-var to signify it and changes logging mode to debug